config/v1/types_scheduling.go

```diff
@@ -28,6 +28,9 @@ type SchedulingSpec struct {
 }
 
 type SchedulingStatus struct {
+	// defaultNodeSelector helps set the cluster-wide default node selector to restrict pod placement to specific nodes.
+	// +optional
+	DefaultNodeSelector string `json:"defaultnodeselector,omitempty"`
```

Comment:
Why is this on status and not spec?  Is a cluster-admin allows to change this?

Reply:
Yeah, it was a mistake, I should have filled in Spec.

Reply:
Yes, cluster-admin should be able to change this.